[site] Added pprof doc (#1003)
kubernetes-sigs/kueue: site/content/en/docs/installation/_index.md

```sh
make undeploy
```

## Enabling pprof endpoints

> _Available in Kueue v0.5.0 and later_

> It's available only if `pprofBindAddress` is set in the [manager's configuration](/docs/installation/#install-a-custom-configured-released-version).

Reviewer: ```suggestion
To enable pprof endpoints, you need to set a `pprofBindAddress` in the [manager's configuration](/docs/installation/#install-a-custom-configured-released-version).
```

> The easiest way to reach pprof port in kubernetes it's to use `port-forward` command:

Reviewer: ```suggestion
The easiest way to reach pprof port in kubernetes is to use `port-forward` command:
```

```shell

> kubectl port-forward kueue-controller-manager-769f96b5dc-87sf2 -n kueue-system 8082:8082

Reviewer: Explain how to get the pod name

Forwarding from 127.0.0.1:8082 -> 8082
Forwarding from [::1]:8082 -> 8082
```

The HTTP endpoint will now be available as a local port.

> Please see [basic usage](https://github.com/google/pprof#basic-usage) pprof command and [examples](https://pkg.go.dev/net/http/pprof#hdr-Usage_examples) of using endpoints

Reviewer 2: ```suggestion
Please see [basic usage](https://github.com/google/pprof#basic-usage) pprof command and [examples](https://pkg.go.dev/net/http/pprof#hdr-Usage_examples) of using the exposed endpoints.
```

> Please see [basic usage](https://github.com/google/pprof#basic-usage) pprof command and [examples](https://pkg.go.dev/net/http/pprof#hdr-Usage_examples) of using the exposed endpoints.

Reviewer: ```suggestion
To learn how to use the exposed endpoint, see [pprof basic usage](https://github.com/google/pprof#basic-usage) and [examples](https://pkg.go.dev/net/http/pprof#hdr-Usage_examples).
```

```sh
make undeploy
```

> ## Enabling pprof endpoints

Reviewer: Also, the installation page is probably not the best place to hold this documentation.

Can you start a new `task` https://kueue.sigs.k8s.io/docs/tasks/

Reviewer: Let's take kubernetes as inspiration https://kubernetes.io/docs/tasks/

I think it's time to divide our tasks in 2 (maybe 3?) subpages:
- Batch Admin
- Batch User

Profiling could be under "Batch Admin", but I wonder if we could add another one: "Platform Engineer/Ops"?

@cortespao any ideas?

Reviewer: I discussed this with some colleagues, and the conclusion is that there are 3 personas, so we can have a subpage for each:

- `Batch Application Developer` (previously known as Batch User).
- `Batch Ops` (previously known as Batch Administrator)
- `Batch Platform Developer` (here goes the pprof doc)

But we can leave this for a follow up.
For now, it suffices to put this in the `tasks` folder and list it under Batch Admin.

Reviewer: I opened https://github.com/kubernetes-sigs/kueue/issues/1007
